arviz/plots/backends/matplotlib/lmplot.py

```diff
@@ -0,0 +1,44 @@
+"""Matplotlib plot linear regression figure."""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from ...hdiplot import plot_hdi
+
+
+def plot_lm(x, y, y_model, y_ppc, num_pp_samples, kind_pp, kind_model):
+    """Matplotlib Linear Regression."""
+    _, ax = plt.subplots(1, 1, figsize=(6, 4), dpi=100)
+
+    # plot data
+    ax.plot(x, y, marker=".", color="C3", lw=0, zorder=10)
+
+    if y_ppc is not None:
+        arr = list(range(y_ppc.draw.size * y_ppc.chain.size))
```

Comment:
```suggestion
        arr = np.arange(y_ppc.draw.size * y_ppc.chain.size)
```

Reply:
Sure, I have made the change in local branch. Actually, currently I am working to add argument `plot_dim`. I will push the changes as soon as I finish it